wordpress-mobile/WordPress-iOS
# This Week Widget

Fixes #NA

To test:
- build/run
- make sure you can install This Week Widgets of both medium and large size
- make sure the data in the widget look consistent with the app

PR submission checklist:
